src/components/UI/BaseButton.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+    <button :style="buttonStyles" :class="buttonClasses">
+        <slot></slot>
+    </button>
+</template>
+
+<script>
+
+export default {
+    name: "BaseButton",
+    props: {
+        fontSize: {
+            type: String,
+            default: "16"
+        },
+        buttonType: {
+            type: String,
+            default: "default"
+        }
+    },
+    computed: {
+        buttonClasses() {
+            return ['button', {
+                'button--primary': this.buttonType === 'primary'
+            }]
+        },
+        buttonStyles() {
+            return { "fontSize": +this.fontSize + "px"};
+        }
+        
+    }
+}
+</script>
+
+<style lang="scss" scoped>
+.button {
+  font-family:'RobotoMono', monospace;
+  font-weight: 500;
+  color: $color-cherry-red;
+  border-radius: 64px;
+  border: 2px solid transparent;
+  padding: 0.8rem 2rem;
```

Comment:
as `font-size` can be set with a prop I wonder if this shouldn't be relative to it (`em` vs `rem`), wdyt?

Reply:
it depends :see_no_evil:
If your intention is for the padding to increase as the size of text inside button increases that might be good idea... Indeed,  it's a good idea. I will follow that :beers: